ignite/handlers/logging.py

```diff
@@ -43,10 +41,10 @@ def log_training_weighted_moving_average(trainer, window_size, weights, history_
 def log_validation_weighted_moving_average(trainer, window_size, weights, history_transform=lambda x: x,
                                            should_log=lambda trainer: True, metric_name="", logger=print):
     if should_log(trainer):
-        total_iterations = len(trainer.validation_data)
-        current_iteration = trainer.current_iteration % total_iterations
+        total_iterations = trainer.total_validation_iterations
+        current_iteration = trainer.current_iteration % (total_iterations + 1)
```

Comment:
I'm not sure the `+1` here is necessary. Do you want to make it start from 1 instead of 0? If that's the case, then the `+1` should be outside the modulo operation

Reply:
I think @fmassa is correct here